site/src/pages/release-notes/0.99.7.mdx

```diff
@@ -0,0 +1,243 @@
+                  .consumerKey("...")
+                  .token("...")
+                  ...
+                  .build();
+  ```
+- The Spring Boot integration now hides all services under `/internal/` for non-management ports when
+  `management.server.port` property is set. #2408 #2502
+- The Spring Boot integration now supports
+  [the new graceful shutdown properties](https://docs.spring.io/spring-boot/docs/2.3.0.RELEASE/reference/html/spring-boot-features.html#boot-features-graceful-shutdown)
+  introduced in Spring Boot 2.3. #2784 #2802
+- Added a new API for handling reference counted or pooled objects such as <type://PooledHttpData>) in
+  a relatively safer way. #2448
+  - See <type://PooledHttpData>, <type://PooledWebClient>, <type://PooledHttpRequest> and
+    <type://PooledHttpResponse> for more information.
+
+## 📈 Improvements
+
+- Cleaned up minor issues reported by [errorprone](https://errorprone.info/). #2772
+- Made some exception messages more user-friendly. #2751
+
+## 🛠️ Bug fixes
+
+- It's now allowed to specify an absolute URL *only when* an <type://WebClient> was created *without* a base
+  URL. #2757
+  ```java
+  WebClient clientWithoutBaseUrl = WebClient.of();
+  WebClient clientWithBaseUrl = WebClient.of("https://example.com/");
+  // Good
+  clientWithBaseUrl.get("/bar");
+  clientWithoutBaseUrl.get("https://foo.com/");
+  // Bad
+  clientWithBaseUrl.get("https://foo.com/");
+  clientWithoutBaseUrl.get("/bar");
+  ```
+- Boolean parameter conversion became more strict in annotated services. #2767 #2774
+  - Only `true`, `false`, `1` and `0` are accepted. Other values will cause a `400 Bad Request` response.
+- <type://DocService> web UI now shows the 'request body' field for `DELETE` and `PATCH` methods in the debug
+  form. #2756 #2819
+- <type://JacksonRequestConverterFunction> now handles the case where the target type has a type parameter,
+  e.g. `List<Long>`. #2769 #2779
+- Fixed a bug where the current <type://ServiceRequestContext> is not pushed when invoking
+  <type://ResponseConverterFunction#convertResponse(ServiceRequestContext,ResponseHeaders,Object,HttpHeaders)>.
+  #2789
+- <type://RequestContextExportingAppender> now handles the `<exports />` tag correctly. #2781
+- Fixed a bug where some Reactive Streams `Subscriber` implementations violate the specification. #2815
+- You no longer get sporadic <type://WriteTimeoutException> from proxied connections. #2801 #2805
+- You no longer get a <type://CancelledSubscriptionException> unnecessarily when using
+  <type://PublisherBasedStreamMessage>. #2797
+- You no longer get sporadic `EncoderException`s from HTTP/1 connections. #2765
+- It's now disallowed to specify the following headers in gRPC `Metadata`. #2718
+  - `:status`
+  - `grpc-message`
+  - `grpc-status`
+  - `armeria.grpc.ThrowableProto-bin`
+
+## 🏚️ Deprecations
+
+- <type://AbstractUnwrappable#delegate()> has been deprecated in favor of <type://Unwrappable#unwrap()>.
+
+## ☢️ Breaking changes
```

Comment:
Should mention the removal of some of the unsafe APIs in favor of the new one.

Reply:
Fixed. Let me know if we need to add more. 🙇